html-css/week1/Ali-Nematollahi/Order Desktop/index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>order-desktop</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css"
+    />
+  </head>
+  <body>
+
+    <section class="layout-container">
+      <!-- Order Summary Part -->
+
+      <div class="gradient container-fluid">
+        <div class="p-4 fs-2">Order Summary</div>
+        <div class="row fs-5 px-4">
+          <div class="col">Total</div>
+          <div class="col text-end">$333 / month</div>
+        </div>
+        <div><hr /></div>
+        <div class="row fs-4 px-4 pb-4">
+          <div class="col">Total</div>
+          <div class="col text-end">$100</div>
+        </div>
```

Comment:
That's a lot of `<div>`. You could use instead a paragraph with 2 span tags inside.
```suggestion
        <p class="row fs-4 px-4 pb-4">
          <span class="col">Total</span>
          <span class="col text-end">$100</span>
        </p>
```